sky/serve/autoscalers.py

```diff
@@ -137,8 +140,9 @@ def evaluate_scaling(self):
             self.scale_up(1)
             self.last_scale_operation = current_time
         elif requests_per_node > self.upper_threshold:
```

Comment:
what if `self.upper_threshold` is `None`, then we shouldn't run the below code right?
we probably should refactor the code a bit to incorporate fix-node policy and autoscaling policy.

Reply:
Sure.. I'll just check if it is `None` for now, then if user only specify `upper_threshold`, that means user only want to autoscale up and #replica will only increase until reach `max_replica`; vise versa for only specify `lower_threshold` case. Only if user specified both upper and lower threshold will enable auto sale up/down. We might need a discussion on here.